v63005/outref/gtm8877.txt

```diff
@@ -0,0 +1,30 @@
+# NO RESTRICTIONS
+original input
+original input
+
+# FILTER RETURNING -1
+%YDB-E-COMMFILTERERR, Error executing the command filter for gtmzsystem. RESTRICTEDOP
+%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: echo 'original input'
+%YDB-E-COMMFILTERERR, Error executing the command filter for gtmpipeopen. RESTRICTEDOP
+%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: echo 'original input'
+ZSTATUS=pipeopenfn+4^gtm8877,%YDB-E-IONOTOPEN, Attempt to USE an I/O device which has not been opened
+
+# FILTER RETURNING A STRING
+filtered output
+filtered output
+
+# FILTER INVOCATION IN A TP TRANSACTION
+filtered output
+filtered output
+
+# FILTER WITH ZSYSTEM AND PIPE_OPEN RESTRICTED
+ZSTATUS=zsystemfn+2^gtm8877,%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: ZSYSTEM
+ZSTATUS=pipeopenfn+3^gtm8877,%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: OPEN PIPE
+
```

Comment:
I would also like a test for the below portion of the release note.

*GT.M reports all filter errors to the operator log accompanied by a COMMFILTERERR.*